net-p2p/nicotine+/nicotine+-3.3.4.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+
+inherit distutils-r1 xdg
+
+DESCRIPTION="Graphical client for the Soulseek peer to peer network written in Python"
+HOMEPAGE="https://nicotine-plus.org/"
+SRC_URI="https://github.com/Nicotine-Plus/nicotine-plus/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/nicotine-plus-${PV}"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+BDEPEND="${PYTHON_DEPS}
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	sys-devel/gettext
+	>=x11-libs/gtk+-3.22.20:3[introspection]
+"
+REPEND="${BDEPEND}"
+
+distutils_enable_tests unittest # used by upstream
```

Comment:
pytest is preferable.

Reply:
pytest is better than unittest. this a case where the difference isnt life or death, but its doesnt mean you should use unittest.

Figure out how to skip problematic tests for example with unittest.

"""
Thirdly, there are cases when a particular feature of a test runner is desired even if it is not strictly necessary to run tests. This is particularly the case with pytest’s output capture that can make test output much more readable with particularly verbose packages.
""""